index.js

```diff
@@ -107,24 +146,32 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
-//
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+function triple(num) {
+  return num * 3
+}
+
+console.log(myMap([1,2,3,4,5], triple)) 
+// => [3,6,9,12,15]
+
+console.log(myMap([2,4,6,8], triple))   
+// => [6,12,18,24]
+console.log("%c----------", "color: red") 
 
 
 // ***** Callbacks - Question 2 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
+console.log("%cCallbacks - Question 2", "color: red")
 
-// console.log(myMap(["Ian", "Chase"], greet)) 
-// // => ["Hi, Ian!", "Hi, Chase!"]
-// console.log("%c----------", "color: red") 
+function greet(name) {
+  return (`Hi, ${name}!`)
```

Comment:
The parentheses around the returned string aren't necessary here, you can just do:

```js
function greet(name) {
  return `Hi, ${name}!`
}
```